### PR TITLE
PCC30

### PR DESCRIPTION
**Pull request by satyamasthanaa**
ATTENTION: before clicking "Create Pull Request" please submit some metadata, thanks!

Difficulty level (1-10): [7]
Estimated time spent (hours): [1]
Completed (yes/no): [yes]

### CODE REVIEW

File: 30/satyam/ts.py [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+            'STRESS TEST THE HOST',
+            'RUN CPU AND MEMORY TESTS SEPARATELY',
+            'RUN MEMTEST',
+            'RTFM',
+            'RETURN TO MINIMUM CONFIG',
+            'REPLACE EVERY DIMM, EVERY CPU AND THE MOTHERBOARD',
+            'CALL AN EXORCIST')
+
+
+def intro_message():
```
Comment (pmayd) [inline 30/satyam/ts.py:16]: I like he use of a function for the inro text!

File: 30/satyam/ts.py [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+    time.sleep(2)
+
+    print("With just one fantabulous spin of THE TROUBLESHOOTING WHEEL OF MISFORTUNE (tm), we'll have you on your way!")
+    time.sleep(4)
+
+def spin():
+    input('\nGive the wheel a spin! (Hit Enter)')
+    print('Spinning...')
+    time.sleep(2)
+    print('\n' + STEPS_LIST[random.randint(0, len(STEPS_LIST) -1)])
```
Comment (pmayd) [inline 30/satyam/ts.py:33]: Try out random.choice! https://docs.python.org/3/library/random.html much cleaner

**Review by pmayd — CHANGES_REQUESTED** [review]
You have pushed two different challenges, 30 and 42, please one PR per challenge

**Comment by pmayd** [comment]
I am still seeing two different files in two different directories so please decide for one challenge and only commit the files for that particular challenge

File: 30/satyam/ts.py (hunk 2 above)
Comment (bbelderbos) [reply 30/satyam/ts.py:33]: thanks @pmayd

**Comment by bbelderbos** [comment]
@satyamasthanaa please open a new PR for challenge #42

**Comment by satyamasthanaa** [comment]
@bbelderbos Sir, I had already made pull request #42 and it has got merged with the main branch.

**Pull request merged**